tests/pytest/test.py

```diff
@@ -929,21 +929,25 @@ def testNumCommandOverflow(env):
     r.expect('JSON.NUMINCRBY', 'big_num', '.', '1.6350000000001313e+308').raiseError()
     r.expect('JSON.NUMMULTBY', 'big_num', '.', '2').raiseError()
     # (value remains)
-    r.assertEqual(r.execute_command('JSON.GET', 'big_num', '.'), '1.6350000000001313e308')
+    # Some serializers emit `e+NN` while others emit `eNN`; both are valid.
+    r.assertEqual(r.execute_command('JSON.GET', 'big_num', '.').replace('e+', 'e'),
```

Comment:
Please open a ticket to get to the bottom of this. We still miss something that might be important.